Dockerfile

```diff
@@ -1,8 +1,13 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER Code Climate <hello@codeclimate.com>
 
 RUN adduser -u 9000 -D app
-USER app
+
+WORKDIR /code
+
+COPY build /usr/src/app
```

Comment:
@wfleming In some earlier commits, for some reason - it didn't work to have

```
COPY build/codeclimate-gofmt /usr/src/app
```
in the dockerfile, with
```
CMD ["/usr/src/app/codeclimate-gofmt"]
```
https://circleci.com/gh/codeclimate-community/codeclimate-gofmt/26

> Step 3 : COPY build/codeclimate-gofmt /usr/src/app
stat /var/lib/docker/btrfs/subvolumes/6cd501cd74c8efd4100d078edaa3e1aa6d90339ac71c1054edf62deb7a2e32a6/usr/src/app/codeclimate-gofmt: not a directory